app/drivers/tools/fuzz/c/DirectedAFLpp.py

```diff
@@ -0,0 +1,171 @@
+import os
+from os.path import join
+
+from app.drivers.tools.fuzz.AbstractFuzzTool import AbstractFuzzTool
+
+
+class DirectedAFLpp(AbstractFuzzTool):
+    def __init__(self):
+        self.name = os.path.basename(__file__)[:-3].lower()
+        super().__init__(self.name)
+        self.image_name = "joern-directed-aflpp"
```

Comment:
Dockerhub image name please or maybe a reference as to how to build it?

Reply:
whoops! forgot to push with the final image name. fixed now ([86192ea](https://github.com/dylanjwolff/cerberus/commit/86192ea620c9f0d4c92a9e32a08a094d3fd91b2a))!